python/ray/serve/_private/logging_utils.py

```diff
@@ -100,7 +100,7 @@ def format(self, record: logging.LogRecord) -> str:
                 The formatted log record in json format.
         """
         record_format = copy.deepcopy(self.component_log_fmt)
-        record_attributes = copy.deepcopy(record.__dict__)
+        record_attributes = record.__dict__
```

Comment:
nit: probably better to just get rid of this aliasing and use `record.__dict__` directly instead

Reply:
great idea, will push a change